src/main/java/Duke.java

```diff
@@ -1,10 +1,105 @@
+import java.util.Scanner;
+import Task.Deadline;
+import Task.ToDo;
+import Task.Events;
+import Task.Task;
+
 public class Duke {
+
+    static Task[] t = new Task[100];
+    static Scanner in = new Scanner(System.in);
+
+    public static void displayTaskList() {
+        System.out.println("    ____________________________________________________________\n" +
```

Comment:
Hi since the separate line is repeatedly used throughout the main method, you may define it as a static constant outside the main method. This could make your code shorter and more readable!